script.js

```diff
@@ -0,0 +1,170 @@
+  },
+  {
+    featuredImage: 'sources/TipicaDeYucalpeten.svg',
+    title: 'Orquesta Tipica de Yucalpeten',
+    description: 'The Orquesta Típica de Yucalpetén is a musical treasure rooted in the cultural tradition of Yucatán, Mexico.',
+    subDescription: 'Each note played by the Orquesta Típica de Yucalpetén resonates with the charm and magic of the Yucatán Peninsula. ',
+  },
+];
+
+function createCards1(artistData) {
+  const cardContent = document.createElement('div');
+  cardContent.classList.add('cardContent');
+
+  const imageContainer = document.createElement('div');
+  imageContainer.classList.add('imageContainer');
+
+  const featuredImage = document.createElement('div');
+  featuredImage.classList.add('featuredImage');
+  featuredImage.innerHTML = `<img src='${artistData.featuredImage}' alt=''>`;
+  imageContainer.appendChild(featuredImage);
+
+  const textContent = document.createElement('div');
+  textContent.classList.add('textContent');
+
+  const title = document.createElement('div');
+  title.classList.add('title');
+  title.innerText = artistData.title;
+
+  const description = document.createElement('div');
+  description.classList.add('description');
+  description.innerText = artistData.description;
+
+  const hr = document.createElement('hr');
+
+  const subDescription = document.createElement('div');
+  subDescription.classList.add('subDescription');
+  subDescription.innerText = artistData.subDescription;
+
+  textContent.appendChild(title);
+  textContent.appendChild(description);
+  textContent.appendChild(hr);
+  textContent.appendChild(subDescription);
+  cardContent.appendChild(imageContainer);
+  cardContent.appendChild(textContent);
+
+  return cardContent;
+}
+
+function createCards2(artistData) {
+  const cardContent2 = document.createElement('div');
+  cardContent2.classList.add('cardContent');
+  cardContent2.classList.add('hide');
+  cardContent2.setAttribute('id', 'hideCards');
+
+  const imageContainer2 = document.createElement('div');
+  imageContainer2.classList.add('imageContainer');
+
+  const featuredImage2 = document.createElement('div');
+  featuredImage2.classList.add('featuredImage');
+  featuredImage2.innerHTML = `<img src='${artistData.featuredImage}' alt=''>`;
+  imageContainer2.appendChild(featuredImage2);
+
+  const textContent2 = document.createElement('div');
+  textContent2.classList.add('textContent');
+
+  const title2 = document.createElement('div');
+  title2.classList.add('title');
+  title2.innerText = artistData.title;
+
+  const description2 = document.createElement('div');
+  description2.classList.add('description');
+  description2.innerText = artistData.description;
+
+  const hr = document.createElement('hr');
+
+  const subDescription2 = document.createElement('div');
+  subDescription2.classList.add('subDescription');
+  subDescription2.innerText = artistData.subDescription;
```

Comment:
- Great Job for rendering speakers dynamically., I think it would be better if you use `article` instead of cards for rendering speaker for sematic reasons 👍